src/get-started/quickstart-py.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: "Beam Quickstart for Python"
+permalink: /get-started/quickstart-py/
+---
+
+# Apache Beam Python SDK Quickstart
+
+* TOC
+{:toc}
+
+## Set up your environment
+
+### Install pip
+
+[Install pip](https://pip.pypa.io/en/stable/installing/), Python's package manager. Check if you already have `pip` installed by running `pip --version`. After installation, verify that you have `pip` version 7.0.0 or newer. To update `pip`, run the following command:
+
+```
+pip install -U pip
+```
+
+### Install virtualenv
+
+It is recommended that you install a [Python virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/)
```

Comment:
I'd simplify -- we don't need to give so many commands. Applies to all steps.

If you don't have XYZ version ABC or newer, install it:
   command to install